src/browser/components/SettingsPage.jsx

```diff
@@ -59,104 +47,6 @@ export default class SettingsPage extends React.Component {
         }
       });
     });
-
-    // when the config object changes here in the renderer process, tell the main process to reload its config object to get the changes
-    config.on('synchronize', () => {
-      ipcRenderer.send('reload-config');
-    });
-
-    // listen for any config reload requests from the main process to reload configuration changes here in the renderer process
-    ipcRenderer.on('reload-config', () => {
-      config.reload();
-    });
-
-    ipcRenderer.on('add-server', () => {
-      this.setState({
-        showAddTeamForm: true,
-      });
-    });
-
-    ipcRenderer.on('switch-tab', (event, key) => {
-      backToIndex(key);
-    });
-
-    ipcRenderer.on('zoom-in', () => {
-      const activeTabWebContents = this.getTabWebContents();
-      if (!activeTabWebContents) {
-        return;
-      }
-      if (activeTabWebContents.getZoomLevel() >= 9) {
-        return;
-      }
-      activeTabWebContents.setZoomLevel(activeTabWebContents.getZoomLevel() + 1);
-    });
-
-    ipcRenderer.on('zoom-out', () => {
-      const activeTabWebContents = this.getTabWebContents();
-      if (!activeTabWebContents) {
-        return;
-      }
-      if (activeTabWebContents.getZoomLevel() <= -8) {
-        return;
-      }
-      activeTabWebContents.setZoomLevel(activeTabWebContents.getZoomLevel() - 1);
-    });
-
-    ipcRenderer.on('zoom-reset', () => {
-      const activeTabWebContents = this.getTabWebContents();
-      if (!activeTabWebContents) {
-        return;
-      }
-      activeTabWebContents.setZoomLevel(0);
-    });
```

Comment:
Redundant since listeners already exist in MainPage component and SettingsPage is now a child of MainPage.